procedures/meeting-observers.md

```diff
@@ -0,0 +1,39 @@
+# Meeting observers
+
+## Motivation
+
+There has been some interest in people joining meetings in a purely observational manner. (See [previous discussion on Zulip](https://rust-lang.zulipchat.com/#narrow/stream/392734-council/topic/Observing.20meetings.3F).) Further discussion on the specifics of a proposal happened on [Zulip](https://rust-lang.zulipchat.com/#narrow/stream/392734-council/topic/Meeting.20observers) and [in a council meeting](https://github.com/rust-lang/leadership-council/blob/245b22a105546a680aad582cf856311a80e93baf/minutes/sync-meeting/2023-07-20.md).
+
+## Considerations
+
+* Unless otherwise necessary, more transparency and openness is usually better
+    * Separate processes (such as releasing minutes or recordings) may fulfill this, but are out of scope for this proposal
+* Any transparency and openness should not negatively affect council processes and procedures, within reason
+    * Public meetings (to any extent) do have the potential for disruption, either through intentional means (harassment of council members) or through unintentional means (less comfort in being able to speak candidly in some situations)
+    * Some overhead for managing access to meetings is expected
+* There will be portions of some meetings that require attendance of only council members, or potentially some set of invited people
+* It may be difficult draw a line between private and public meetings that most satisfies the above points
+
+## Process for meeting observers
+
+### Request for observation
+
+As an initial iteration, any Project member (as currently defined by anyone listed in the 'all' team defined in the team repo) may join the public portion of council meetings to observe. No particular method of communication is required; those interested should either post in the [council zulip stream](https://rust-lang.zulipchat.com/#narrow/stream/392734-council), send an email to `council@rust-lang.org`, or (if necessary) message a council member. At least for now, non-Project members will not be admitted as meeting observers.
+
+### Meeting process
+
+When a meeting is scheduled, those who have requested to observe that meeting will get a calendar invite, in addition to the council. A minutes link will not be shared (since it may contain private information), but a public agenda link may be.
+
+During the meeting, observers should be mostly non-interactive, unless otherwise noted: no chat messages should be sent, and video and audio should be off. However, there may be times, like during check-ins and check-outs, where observers are encouraged to share their experience in chat. Observers should not record or screenshot the meeting. If a meeting has a private session, then observers will either leave or be kicked.
```

Comment:
I think it'll often happen that someone who has a lot of context on a topic we're discussing will be present as an observer. In those situations, I think it'd be very reasonable if we ask that person to give us relevant context or answer questions.

And even if we don't ask them questions, for example if we simply don't realize that someone in the room has more context than us, would we not expect that person to speak up and give us the context? It'd be a very awkward situation for an observer if their effort/feature/thing is discussed and they can't tell us about the information we're missing or got wrong.

Reply:
I think this is a good observation.

I wonder if it's worth having a separate category of invited experts? I imagine there might be times we want to invite someone specifically to have them be ready to provide input on a certain topic.

We could also say observers are allowed to raise their hand, but we may choose not to call on them?

Of course, too much of this could conflict with other goals, like running efficient meetings. For example, if we have observers and invited experts we now have three different categories of attendees with different permissions that the facilitator has to keep track of. Or we have to have a brief interruption to say "I'm sorry, could you put your hand down?"

Maybe it's better to rely on out of band mechanisms for getting expert opinions? Like, the kinds of discussion that benefit from outside expert input are probably best had on Zulip in the public channel.

Reply:
I have added the following paragraph below to address this:
>It is expected that, by default, those joining as meeting observers will follow the above guidelines. However, there may be times where the council will invite guests to participate beyond the role and guidelines of meeting observers. This may happen ahead of the meeting time or, at the discretion of the meeting participants, during the meeting itself. Under these circumstances, adherence to meeting observer guidelines is not expected.